index.php

```diff
@@ -53,15 +83,12 @@
 		<div class="news medium"></div>
 	</div>
 
-	<div class="farbottom center-hor">
+	<!-- <div class="farbottom center-hor">
 		<div class="newsDots xxxsmall"></div>
-	</div>
+	</div> -->
 
-	<div class="farbottom right">
```

Comment:
I actually use these for debugging.  It should probably be a config.js option, as should tweecool.